lib/utilities.h

```diff
@@ -69,6 +164,32 @@ namespace gr {
             return (count & 0x1) == !even;
         }
 
+        /**
+         *  \brief  Check whether the parity of the given uint64_t is even.
+         *
+         *  \param  word
+         *          The uint64_t to check.
+         *  \param  even
+         *          Check for even (`true`) or uneven (`false`) parity.
+         */
+        bool check_parity(uint64_t word, const bool even = true) {
```

Comment:
Should be credited to source: https://graphics.stanford.edu/~seander/bithacks.html